source/code/include/playfab/PlayFabIXHR2HttpPlugin.h

```diff
@@ -1,3 +1,4 @@
+#if defined(PLAYFAB_PLATFORM_XBOX)
 #pragma once
```

Comment:
You may move this pragma higher, out of the protection block

Reply:
👍